cephadm-ansible.spec.in

```diff
@@ -0,0 +1,42 @@
+%global commit @COMMIT@
+%global shortcommit %(c=%{commit}; echo ${c:0:7})
+
+Name:           cephadm-ansible
+Version:        @VERSION@
+Release:        @RELEASE@%{?dist}
+Summary:        Ansible playbooks for Ceph
+License:        ASL 2.0 and GPLv3+
+URL:            https://github.com/ceph/cephadm-ansible
+Source0:        %{name}-%{version}-%{shortcommit}.tar.gz
+
+BuildArch:      noarch
+
+BuildRequires: ansible >= 2.10
+Requires: ansible >= 2.10
+
+%description
+Ansible playbooks for Ceph
```

Comment:
Let's change this description to match whatever goes into the README

Reply:
README update in #2 says "cephadm-ansible is a collection of Ansible playbooks to simplify workflows that are not covered by cephadm."